### DISCUSSION TITLE
Improve text color for certain label colors

### DISCUSSION THREAD

**Discussion by Andre601** [badges/shields#8186] [Ideas] [answered]
Certain colours make the text on the right harder to read. This is the case for `green` where the white font is a lot more difficult to see than with other colours.

Shields.io should have a system that would adjust the font colour if the colour used would be too bright for the normal white colour.
Alternatively (Or in addition to it) could shields.io offer a query option to override the font color for the left and right side of the badge.

Here's an example displaying the issue with the bright color:
![image](https://user-images.githubusercontent.com/11576465/178219889-969d2133-1487-4998-8bce-51b6bc5641a6.png)

**Comment by Andre601** [comment]
Well, it apparently does already change font colour (i.e. for colour `9cf`), tho it seems that green (And a few others) don't count towards this.

Colors I noticed to have hard to read text:

- ![green](https://img.shields.io/badge/Color-Green-green)
- ![brightgreen](https://img.shields.io/badge/Color-Bright%20Green-brightgreen)
- ![yellow](https://img.shields.io/badge/Color-Yellow-yellow)
- ![lightgrey](https://img.shields.io/badge/Color-Light%20Grey-lightgrey)

**Comment by calebcartwright** [comment, marked as answer]
Relevant backstory and reasoning in #5467 and https://github.com/badges/shields/pull/5467#issuecomment-683924571 and #5497
